src/app/legacy/containers/PageHandlers/withContexts.jsx

```diff
@@ -15,25 +11,25 @@ const WithContexts = Component => {
   const WithContextsContainer = props => {
     const {
       toggles,
-      bbcOrigin,
-      status,
-      id,
+      bbcOrigin = null,
+      status = null,
+      id = null,
       service,
       isAmp,
       isApp,
       isLite,
       isCaf,
```

Comment:
Do these also need to be defaulted to `false` per the `defaultProps` below?

```suggestion
      isLite = false,
      isCaf = false,
```

Reply:
Good spot, will add